### DISCUSSION TITLE
Import data (Firefox (csv)) not using "timePasswordChanged"

### DISCUSSION THREAD

**Discussion by 3ftomi** [dani-garcia/vaultwarden#2408] [Q&A]
### Subject of the issue

Exported login credentials from Firefox (latest:99) to .csv. In the .csv there is a row named "timePasswordChanged".
I supposed this field used to the "Updated" time field in Vaultwarden. Unfortunately I've faced that all Items have the same "Updated" time which is the time when the importing processed and not the original time (from the .csv) when a login changed.

This date and time are important to know how old it is. Maybe needed to change. Or if there is multiple instances which one is the newer.

### Deployment environment

* vaultwarden version: 1.24.0

* Install method: Docker image (QNAP, Container Station)

* Clients used: web vault

* Reverse proxy and version: QNAP QTS 5.0.0.1986 Builtin Reverse Proxy

* MySQL/MariaDB or PostgreSQL version: MariaDB 5.5.68

* Other relevant details:

### Steps to reproduce

In Firefox > about:logins > upper right corner three dots menu > Export logins... > created logins.csv file
In Vaultwarden web vault > Tools menu > Import data > Firefox (csv) > Import data

### Expected behaviour
In My vault > Item > "Updated:" time use the time when last changed (from logins.csv "timePasswordChanged" field)

### Actual behaviour
In My vault > Item > "Updated:" time now showing the time when import taken.

**Comment by BlackDex** [comment]
That information is not sent by the clients and thus not something we can use or implement.
Also, this is something which is specifically build into the clients, which the Vaultwarden team does not maintain.

I would suggest to check if this also happens on vault.bitwarden.com (which i think it is) and then post a feature request here: https://community.bitwarden.com/c/feature-requests/5
